src/manifest/dbtProject.ts

```
@@ -713,11 +884,70 @@ export class DBTProject implements Disposable {
+    switch (dbtIntegrationMode) {
+      case "cloud":
+      case "fusion":
+        // For cloud/fusion integrations, validate authentication
+        try {
+          this.validationProvider.validateCredentialsSilently();
+          return true;
+        } catch (e) {
+          window.showErrorMessage((e as Error).message);
+          return false;
+        }
+      case "core":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)